src/averageAges.js

```diff
@@ -37,7 +42,16 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = withChildren
+    ? people.filter((p) => p.sex === 'f'
+      && people.find((p1) => p1.mother === p.name))
```

Comment:
Ми ж тут шукаємо дитину, тому давай напишемо саме це. І для більш читабельності коду не скорочуй так сильно змінні
```suggestion
      && people.find((сhild) => child.mother === p.name))
```